src/main/java/base/BaseTest.java

```diff
@@ -10,7 +10,7 @@
 
 public class BaseTest extends BasePage {
 
-    protected WebDriver driver;
```

Comment:
de ce ati schimbat modificatorul de acces pentru Webdriver driver?

Reply:
am schimbat in protected